### PR TITLE
Outguess 0.2 (new formula)

### PR DESCRIPTION
**Pull request by ryanpconnors**
Adds formula for outguess steganographic tool.

### CODE REVIEW

File: Formula/outguess.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  url "https://ftp.mirrorservice.org/sites/ftp.wiretapped.net/pub/security/steganography/outguess/outguess-0.2.tar.gz"
+  sha256 "2f951ed7b9b9373fae8fe95616d49c83ae246cf53a2b60a82814228515bfa7d6"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make"
+    bin.install "outguess"
```
Comment (DomT4) [inline Formula/outguess.rb]: No `make install`?

**Comment by DomT4** [comment]
Apparently there's already a [Cask](http://www.rbcafe.com/install-outguess-with-brew/) for [this](https://github.com/caskroom/homebrew-cask/blob/881b3e0ce869b37cc1848cd80bf3d9bdaaac279e/Casks/outguess.rb). Is there any advantage to building from source?

**Comment by ryanpconnors** [comment]
@DomT4 the advantage is the command line tool. The Cask is for the GUI version of outguess.

File: Formula/outguess.rb (hunk 1 above)
Comment (ryanpconnors) [inline Formula/outguess.rb]: I tried `make install` but got the following error:
```
Last 15 lines from /Users/Ryan/Library/Logs/Homebrew/outguess/02.make:
clang -pipe -Wall -funroll-all-loops -O2  -I.   -c -o jdmerge.o jdmerge.c
clang -pipe -Wall -funroll-all-loops -O2  -I.   -c -o jcomapi.o jcomapi.c
clang -pipe -Wall -funroll-all-loops -O2  -I.   -c -o jutils.o jutils.c
clang -pipe -Wall -funroll-all-loops -O2  -I.   -c -o jerror.o jerror.c
clang -pipe -Wall -funroll-all-loops -O2  -I.   -c -o jmemmgr.o jmemmgr.c
clang -pipe -Wall -funroll-all-loops -O2  -I.   -c -o jmemnobs.o jmemnobs.c
rm -f libjpeg.a
ar rc libjpeg.a  jcapimin.o jcapistd.o jctrans.o jcparam.o jdatadst.o jcinit.o jcmaster.o jcmarker.o jcmainct.o jcprepct.o jccoefct.o jccolor.o jcsample.o jchuff.o jcphuff.o jcdctmgr.o jfdctfst.o jfdctflt.o jfdctint.o jdapimin.o jdapistd.o jdtrans.o jdatasrc.o jdmaster.o jdinput.o jdmarker.o jdhuff.o jdphuff.o jdmainct.o jdcoefct.o jdpostct.o jddctmgr.o jidctfst.o jidctflt.o jidctint.o jidctred.o jdsample.o jdcolor.o jquant1.o jquant2.o jdmerge.o jcomapi.o jutils.o jerror.o jmemmgr.o jmemnobs.o
ranlib libjpeg.a
clang -pipe -Wall -funroll-all-loops -g -O2 -Imissing -I./jpeg-6b-steg  -o outguess outguess.o golay.o arc.o pnm.o jpg.o iterator.o md5.o  -L./jpeg-6b-steg -ljpeg -lm
ln -sf outguess extract
/usr/bin/install -c -m 755 outguess /usr/local/Cellar/outguess/0.2/bin
/usr/bin/install -c -m 644 outguess.1 /usr/local/Cellar/outguess/0.2/man/man1
install: /usr/local/Cellar/outguess/0.2/man/man1: No such file or directory
make: *** [install] Error 71

READ THIS: https://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/brew):
  https://github.com/Homebrew/homebrew-core/issues
```

Comment (MikeMcQuaid) [inline Formula/outguess.rb]: Please report that issue to the upstream developers and add a `man1.mkpath` before it for now.

File: Formula/outguess.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    man1.install "outguess.1"
+  end
+
+  test do
+    `outguess`
```
Comment (MikeMcQuaid) [inline Formula/outguess.rb]: `assert_match "something", shell_output("outguess", 1)`

File: Formula/outguess.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    assert $?.exitstatus == 1
```
Comment (MikeMcQuaid) [inline Formula/outguess.rb]: And then can remove this line.

**Comment by DomT4** [comment]
Closing as seems to have gone dead here. Happy to take a fresh PR with the various comments addressed. Thank you nonetheless for your submission to Homebrew ❤️

**Pull request closed without merging**